.travis.yml

```diff
@@ -38,15 +38,14 @@ before_install:
 
 install:
   - composer update --prefer-source -o $COMPOSER_FLAGS
+  - if [[ ${TRAVIS_PHP_VERSION:0:3} == "7.1" ]]; then wget http://cs.sensiolabs.org/download/php-cs-fixer-v2.phar -O $(pwd)/php-cs-fixer; fi
```

Comment:
Please remove the pwd, shouldn't be necessary since we aren't changing the working directory